packages/adminrouter/extra/src/includes/server/master.conf

```diff
@@ -17,13 +17,28 @@ location /pkgpanda/active.buildinfo.full.json {
 
 # Group: Root
 # Description: DC/OS GUI
-root /opt/mesosphere/active/dcos-ui/usr;
+root /opt/mesosphere/active/dcos-ui-service/dist/ui;
```

Comment:
I wonder if we need regenerating `packages/adminrouter/extra/src/docs/api/nginx.master.yaml`